docs/GettingStarted.md

```diff
@@ -2135,17 +2135,24 @@ LogJob.perform_async('login')
 
```

Comment:
You mention in the PR description:

> This is part of https://github.com/DataDog/dd-trace-rb/pull/2848, and is being merged into a feature branch.

To be honest, this seems kinda harmless to merge directly to master to avoid having an even bigger feature branch that needs to be updated, rebased, etc.

Just a suggestion from my side, since this technique is what I almost always end up doing ;)

Reply:
Into master we go!